src/app/design/guidelines/back-button/index.html

```diff
@@ -0,0 +1,71 @@
+<stache
+  pageTitle="Back Button"
+  navTitle="Back button"
+  showTableOfContents="true">
+
+  <stache-page-anchor>
+    New page behavior
+  </stache-page-anchor>
+
+  <p>
+    When users navigate to a new context such as a new record, list or section, the action should create an entry in the browser history with a new page title. This entry enables users to select the browser's back button to return to the previous page or view.
+  </p>
+  <p>
+    As a rule of thumb, create a new page in the browser history if the majority of the page content changes or users perceive a signficant change of context. Don't create a new URL or new browser entry for an inline interaction such as steps in a wizard or opening a detail panel.
+  </p>
+
+  <div class="sky-subsection-heading">Focus</div>
+
+  <p>
+    After loading a new page, users should be able to use the tab key to reach the first focusable element on the page.
+  </p>
+
+  <div class="sky-subsection-heading">Screen readers</div>
+
+  <p>
+    When a page loads, it should behave similarly to what users expect from a regular server-side application. The page should announce the page's <sky-code>&lt;title&gt;</sky-code> to the screen reader and inform users that it loaded via an ARIA live region when the <sky-code>&lt;title&gt;</sky-code> element is ready. Behaviors vary based on different browser and screen reader combinations, but SKY UX applications use this combination.
```

Comment:
@blackbaud-johnly @Blackbaud-ToddRoberts Has this functionality been added to SKYUX? This proposal was to add a common solution to the framework so team's wouldn't have to do this one off. If it hasn't been added, then I would wait to include this. Also, is there an issue for it in the backlog?

Reply:
Actually just found there is an issue and this isn't built yet. I think we should remove this until we offer a way for the system to actualize it. https://github.com/blackbaud/skyux-core/issues/67